src/components/OverlayConfigDialog.js

```
@@ -0,0 +1,207 @@
+    const state = store.getState();
+    const mapping = { ...state.mapping };
+    mapping[this.field] = [...(mapping[this.field] || [])];
+    const base = { ...mapping[this.field][this.index] };
+
+    if (isStepMode) {
+      const dy = parseInt(rowVal, 10);
+      const dx = parseInt(colVal, 10);
+      base.dy = Number.isFinite(dy) ? dy : 1;
+      base.dx = Number.isFinite(dx) ? dx : 0;
+      delete base.step;
+      delete base.script;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)